src/extension.ts

```diff
@@ -1097,6 +1098,24 @@ async function getLogsForPod(pod: PodSummary, follow?: boolean) {
     }
 }
 
+async function getEvents(follow: kuberesources.EventDisplayMode, explorerNode?: explorer.ResourceNode) {
```

Comment:
rename the `follow` variable, since it doesn't describe the other condition (show).

Reply:
thought to do it but somehow got slipped out of my mind